editor/src/document/document_file.rs

```diff
@@ -694,7 +706,12 @@ impl MessageHandler<DocumentMessage, &InputPreprocessor> for DocumentMessageHand
 					.into(),
 				);
 			}
-
+			ReRenderDocument => {
+				//When the document is zoomed the wireframes must be re-rendered in order to maintain wireframe stroke width
+				if self.graphene_document.view_mode() == ViewMode::WireFrame {
+					GrapheneDocument::visit_all_shapes(&mut self.graphene_document.root, &mut (|_s: &mut GrapheneShape| {}));
+				}
+			}
```

Comment:
This seems weird to me, I would expect something like this:
1. Receive SetViewMode Message
2. Send operation to recursively apply view mode
3. Apply view mode in the document
4. Send RenderDocument Message